packages/react-ui/src/app/features/home/flows-table/flows-table.tsx

```diff
@@ -53,9 +53,10 @@ const HomeFlowsTable = ({
           loading={loading}
           stickyHeader
           border={false}
+          getRowHref={(row) => `/flows/${row.id}`}
           onRowClick={(row, e) => {
-            if (e.ctrlKey) {
-              window.open(`/flows/${row.id}`, '_blank');
+            if (e.ctrlKey || e.metaKey || e.button === 1) {
```

Comment:
can extract this common logic to a helpers file (for example: `table-navigation-helper.ts`, also magic number can be extracted as constant, like:
```js
const MiddleMoustButtonClickEvent = 1;
```